src/content/structured/components/data-table/accessibility.mdx

```diff
@@ -0,0 +1,81 @@
+---
+path: "/components/data-table/accessibility"
+
+date: "2025-05-14"
+
+title: "Data table"
+
+status: "CANARY"
+
+subTitle: "Use a data table to organise and display large quantities of related data."
+
+tabs:
+  [
+    { title: "Guidance", path: "/components/data-table" },
+    { title: "Code", path: "/components/data-table/code" },
+    { title: "Accessibility", path: "/components/data-table/accessibility" },
```

Comment:
These tabs need to be copied to the code page also, the a11y tab is not accessible via that page.